Runtime/Rigs/XRSimulatorSetup.cs

```diff
@@ -0,0 +1,36 @@
+﻿#if XRIT_1_0_OR_NEWER
```

Comment:
Do you think it's good to completely "hide" the simulator instead of giving as setup tooltip hint that you need version 1.0 or higher to run the simulator?